Haddock failure: Network/HTTP2/Server/Types.hs:33:5: error: parse error on input ‘}’

I get the following error when building haddocks:

```
Network/HTTP2/Server/Types.hs:33:5: error: parse error on input ‘}’
   |
33 |     }
   |     ^
```

Caused by this line: https://github.com/kazu-yamamoto/http2/blob/136188cdf4c2d750f3f8ac780a7be4b33b36d12d/Network/HTTP2/Server/Types.hs#L32

--- a/Network/HTTP2/Server/Types.hs
+++ b/Network/HTTP2/Server/Types.hs
@@ -29,7 +29,6 @@ data PushPromise = PushPromise {
       promiseRequestPath :: ByteString
     -- | Accessor for response actually pushed from a server.
     , promiseResponse    :: Response
-    -- | Accessor for response weight.
     }
 
 -- | Additional information.

